README.md

```diff
@@ -223,3 +223,22 @@ This will run `eslint --fix` and automatically add changes to the commit. Please
   ]
 }
 ```
+
+### In-place minify the images before adding to the git repo
+
+```json
+{
+  "*.{png,jpeg,jpg,gif,svg}": [
+    "imagemin-lint-staged",
+    "git add"
+  ]
+}
+```
+
+<details>
+  <summary>More about `imagemin-lint-staged`</summary>
```

Comment:
Markdown isn't supported inside `<summary>` tag. Use `<code></code>` instead.

Reply:
Got it. Will do